guides/migrate-to-v4.md

```diff
@@ -0,0 +1,33 @@
+# Migrating from @celo-tools/use-contractkit
+
+This guide covers moving from @celo-tools/use-contractkit@3.x to @celo/react-celo@4.x
+
+Do Not be thrown thrown off by the name change. This is the same code base (with a few [breaking changes](#link-to-release-notes)).
+
+## Step 1
+
+Remove @celo-tools/use-contractkit
+
+`yarn remove @celo-tools/use-contractkit`
+
+or
+
+## Step 2
+
+Install @celo/react-celo and at least version 2 of @celo/contractkit
+
+`yarn add @celo/react-celo @celo/contractkit@latest`
+
+_If your app directly imports any other @celo sdks such as @celo/utils update them as well_
```

Comment:
Update to latest, to v4, or to which version? What's the minimum version they need to update the sdks to? 
For this document to be evergreen, it would be useful to specify what's the SDK version that is compatible with `react-celo@v4`.